CMakeLists.txt

```diff
@@ -26,6 +26,16 @@ include(GNUInstallDirs)
 include(FortranCInterface)
 include(CMakePackageConfigHelpers)
 
+# Requires OpenMP
```

Comment:
If those libraries are called from threaded code and aren't compiled already with OpenMP, then yes, they should have the recursive flag. I think some of those are compiled with OpenMP already?

For the second question, that might be the safest thing to do. We could *probably* get away with not using the thread-safe version (for erd at least), but it might be easier to just always use the threaded version.